src/ensembl/src/content/app/entity-viewer/gene-view/GeneView.tsx

```diff
@@ -361,7 +361,17 @@ const useGeneViewRouting = () => {
 
   useEffect(() => {
     if (view && viewInRedux !== view) {
-      dispatch(updateView(view as View));
+      if (view in View) {
```

Comment:
I've searched on StackOverflow for how this is supposed to work, and the [answer](https://stackoverflow.com/a/43805090/3925302) is confusing. Are we sure we even want an enum here? Would just a simple [array of strings](https://thoughtbot.com/blog/the-trouble-with-typescript-enums#another-option-since-typescript-34) work better for us? Just curious.

Reply:
Yes agree that was overkill and also it wasn't working properly. I've changed this in df97ef9

Reply:
✅